documentation/deployments.md

```diff
@@ -11,3 +11,57 @@ Upon merging a PR to `main` a deployment to the development environment will be
 1. Initialise Terraform
 1. Run the Terraform plan
 1. Apply the Terraform changes
+
+### PaaS
+
+Some useful things to know about PaaS...
+
+GOV.UK Platform as a Service (PaaS) is a cloud-hosting platform built by the Government Digital Service (GDS). GOV.UK PaaS manages the deployment of your apps, services and background tasks so you don’t need to hire people with specialist cloud skills.
+
+GOV.UK PaaS uses the open source Cloud Foundry project, and runs on Amazon Web Services.
+
+This repo has an organisation setup on PaaS. Speak to the org manager to authorise the [creation of a personal account](https://docs.cloud.service.gov.uk/get_started.html#get-started)
+
+The Cloud Coundry CLI can be used to manage apps hosted on PaaS - [Setup Cloud Foundry CLI](https://docs.cloud.service.gov.uk/get_started.html#set-up-the-cloud-foundry-command-line)
+
+### Environments
+
+This application runs through four environments:
+
+- dev
+- test
+- pre prod
+- prod
+
+`dev` is deployed to automatically when a PR is merged into the `main` branch.
+
+`test`, `pre prod` and `prod` deployments are triggered manually from the Github workflow tab
+
+Each environment has a dedicated PaaS user account to run CI/CD jobs using Github workflows.
+
+To find out the user account details, contact the org manager.
+
+### Seed data
+
+[Seed data is destroyed](../db/seeds.rb) and recreated in `dev`, `test` and `pre prod` environments.
+
+`prod` does not get seeded with data automatically.
+
+### Environment variables
+
+The application requires the following environment variables to deploy to an environment:
+
+- TF_VAR_paas_app_docker_image
+- TF_VAR_paas_user
+- TF_VAR_paas_password
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
+### Databases
+
+The application is backed by a [PostgreSQL database hosted on PaaS](https://docs.cloud.service.gov.uk/deploying_services/postgresql/#postgresql).
+
```

Comment:
i think we should mention that the application stores assets separately with s3, so that future developers wont expect to find these assets stored in the PAAS filesystem

Reply:
This should probably be in the main README.md docs